make/config/kyverno/kustomization.yaml

```diff
@@ -9,16 +9,24 @@ resources:
   - https://github.com/kyverno/policies/pod-security/enforce
   - https://raw.githubusercontent.com/kyverno/policies/main/other/res/restrict-automount-sa-token/restrict-automount-sa-token.yaml
   - https://github.com/kyverno/policies/raw/main//best-practices/require-ro-rootfs/require-ro-rootfs.yaml
+
 patches:
-  - patch: |-
-      - op: replace
-        path: /kind
-        value: Policy
-      - op: add
-        path: /metadata/namespace
-        value: cert-manager
```

Comment:
I removed this patch, because I now want to keep the original ClusterPolicy resources which will catch the HTTP01 Pod in any of the `e2e-tests-` namespaces.